src/lib/change-email-helpers.js

```diff
@@ -0,0 +1,92 @@
+const helpers = require('../lib/helpers');
+const uuid = require('uuid/v4');
+const moment = require('moment');
+const { pool } = require('../lib/connectors/db');
+const { EmailChangeError, checkEmailAddressInIDM } = require('../controllers/change-email');
+const Repository = require('@envage/hapi-pg-rest-api/src/repository');
+
+const usersRepo = new Repository({
```

Comment:
It might be neater to extend the repository base class for each table.  E.g.
```
class UsersRepository extends Repository {
findById(userId) {
...
```